src/accelerators/SinRotAccelerator.h

```diff
@@ -0,0 +1,81 @@
+#pragma once
+
+#include "wiring.h"
+
+#include <cmath>
+#include <cstdint>
+#include <algorithm>
+
+class SinRotAccelerator
```

Comment:
@frmdstryr : This is just a copy of my linear accelerator. Would be great if you could replace that with your sinus accelerator.